api_tests/providers/registrations/views/test_registration_provider_schemas.py

```diff
@@ -79,6 +85,14 @@ def provider_with_egap_only(self, egap_schema):
         provider.save()
         return provider
 
+    @pytest.fixture()
+    def provider_with_prereg(self, osf_prereg_schema, egap_schema, schema):
```

Comment:
including the egap schema here as a param but not adding it to the supported schemas. Adding it *should* break the test as currently constructed